app/assets/javascripts/angular/controllers/pages/recipe_finder_controller.js.erb

```diff
@@ -155,14 +128,21 @@ app.controller('RecipeFinderCtrl', ['$scope', '$http', '$routeParams', function(
     }
   };
 
+  $scope.searchWord = "Ingredient";
```

Comment:
Why do you set ` $scope.searchWord = "Ingredient";` and check the condition later on `if ($scope.searchWord == "Ingredient") {`?